femitj/my-reads
# #168543409 Setup React Environment

#### What does this PR do?

Setup React Environment
  - Sets up babel and webpack
 - Sets up eslint and prettier

#### Description of Task to be completed?

* Install react, react-dom
* Install babel and webpack packages
* Install eslint, eslint-config-airbnb-base and eslint-plugin-import packages
* Add eslint configuration file
* Installs prettier

#### How should this be manually tested?

* From the cli, Run npm run start:dev
* Install vscode [eslint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) extension 
* Go to ```index.js``` file in the ```src``` directory and declare a random variable
* You should see a ```variable is declared but it's value is never read``` error

#### Any background context you want to provide?
N/A

#### Screenshots (if appropriate)
N/A

#### PT-ID

[#168543409](https://www.pivotaltracker.com/story/show/168543409)
